src/connector/src/sink/kafka.rs

```diff
@@ -477,7 +477,9 @@ impl KafkaSinkWriter {
             .await?;
             let producer_ctx = RwProducerContext::new(ctx_common);
             // Generate the producer
-            c.create_with_context(producer_ctx).await?
+            c.set_log_level(read_kafka_log_level())
```

Comment:
I think we can set sth like `RUST_ENV="librdkafka=trace"`, why need a separate env var? :eyes: